source/extensions/config_subscription/grpc/grpc_stream_interface.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "source/common/grpc/typed_async_client.h"
+
+namespace Envoy {
+namespace Config {
+
+// Oversees communication for gRPC xDS implementations (parent to both regular xDS and delta
+// xDS variants). Reestablishes the gRPC channel when necessary, and provides rate limiting of
+// requests. This interface is for a specific discovery Request and Response types.
```

Comment:
what does "specific discovery Request and Response types" mean here exactly?

Reply:
Agree that this is confusing, removed.